Python/Translator_and_Language_Detector/Language Detection and Translation.py

```diff
@@ -0,0 +1,168 @@
+from IPython.display import Image
+Image ("https://specials-images.forbesimg.com/imageserve/1183427805/960x0.jpg")
+
+from googletrans import Translator, constants
+from pprint import pprint
+
+
+translator = Translator()
+
+
+
+# translate a Indonesia text to english text (by default)
```

Comment:
Interesting, your profile seems Indian,  am I supposed to believe that Indonesian is your first language that's why you have made it default?